pkg/daemon/ceph/mds/filesystem.go

```diff
@@ -107,11 +104,13 @@ func (f *Filesystem) CreateFilesystem(context *clusterd.Context, clusterName str
 	return nil
 }
 
-// Remove the file system in ceph
+// DeleteFilesystem removes the filesystem in Ceph and removes the Ceph file daemons.
 func DeleteFilesystem(context *clusterd.Context, clusterName, filesystemName string) error {
 	logger.Infof("Removing file system %s", filesystemName)
 
 	// mark the cephFS instance as cluster_down before removing
+	// TODO: is it safe to proceed immediately after setting the filesystem down, or should Rook
```

Comment:
is this TODO needed? If we're deleting the filesystem, what are you suggesting needs to propagate?

Reply:
I was wondering if the 'cluster_down' state is effective (basically) immediately or if Rook should wait for the filesystem to reach some sort of 'cluster_down' state before proceeding to delete things. I guess for deleting, it doesn't matter since it's going to be deleted anyway. I'll remove this.